### DISCUSSION TITLE
blog/integrating-adminlte-with-aspnet-core/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#33] [General]
# blog/integrating-adminlte-with-aspnet-core/

In this article, we will learn about Integrating AdminLTE with ASP.NET Core 3.1 MVC or really any other Bootstrap based UI Frameworks completely from scratch. We will also go through about integrating Identity Authentication to our MVC Applicaiton. Also, you will gain quite a lot of practical knowledge on Views, Layouts, Partial Views, Conditional Rendering, Navigation Indicatior and much more.

https://codewithmukesh.com/blog/integrating-adminlte-with-aspnet-core/

**Comment by kmorales026** [comment]
Hello!, Nice and amazing post!!!,

**Comment by dpi627** [comment]
What a great job!!
I'm integraing AdminLTE with ASP.NET Core 8 MVC.
This article is really helpful for me, thanks.

**Comment by francogiacobbi** [comment]
Hello, great job and thanks for this.
If can help:
I had to add a View file in Views Folder called _ViewImports.cshtml and add to this file 
 @addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers

otherwise partial view not working.

I've to migrate a WebApp written in Asp.net 4.6 to Asp.Net Core and i'm trying to adapt it with AdminLTE and Angular.

Thanks again.

**Comment by Mahbub-Hossain-Durjoy** [comment]
This article is really helped me a lot. I have learned a lot and successfully integrated the theme.
